### ISSUE TITLE
Prevent orphan node warnings on export

### ISSUE THREAD

**Issue by snoopdouglas** [samsface/godot-steam-api#8]
When exporting, I'm finding that Godot spits out a lot of warnings saying that `steam.gd` has been orphaned, a la:

```
WARNING: cleanup: ObjectDB instances leaked at exit (run with -
-verbose for details).
```

It appears this can be fixed by changing `_exit_tree` to `disable_plugin` in `plugin.gd`.

**Comment by samsface** [comment]
Change the cleanup to be in disable_plugin and don't see any warnings anymore :pray:

**Comment by snoopdouglas** [comment]
Awesome. Fixed in d9143f2fbdf9327556969eb98e0d4fae1bbddeb9
